packages/docusaurus/src/server/getHostPort.ts

```diff
@@ -99,5 +99,6 @@ export async function getHostPort(options: HostPortOptions): Promise<{
   const host = options.host ?? 'localhost';
   const basePort = options.port ? parseInt(options.port, 10) : DEFAULT_PORT;
   const port = await choosePort(host, basePort);
+
```

Comment:
not needed?

Reply:
Are you talking about the basePort L100 ? It was like this originally we may get rid of it

Reply:
I'm talking about the useless extra line break added there: if possible reduce the diff to the minimum

```suggestion
```